integrations-and-supported-tools/detectron2/scripts/requirements.txt

```
@@ -0,0 +1,4 @@
+neptune-client
+neptune-detectron2
+opencv-python==4.5.5.64
```

```suggestion
opencv-python
```
Will revert if this causes tests to fail